src/component/feature/FeatureView2/FeatureVariants/FeatureVariantsList/AddFeatureVariant/AddFeatureVariant.tsx

```diff
@@ -204,6 +208,17 @@ const AddVariant = ({
         ]);
     };
 
+    const isJSON = jsonString => {
+        try {
+            let parsedString = JSON.parse(jsonString);
+            if (parsedString && typeof parsedString === 'object') {
```

Comment:
Unsure whether we'll allow it, but this is an example from MDN on JSON.parse: 

```
JSON.parse('"foo"')
```

It is technically valid JSON and is typeof string.

Reply:
I think we want to allow any valid JSON on further reflection. Then the check here is uneccesary because JSON.parse will throw if it is not valid.